contributors.yml

```
@@ -402,4 +403,8 @@
 - yuleicul
 - zeromask1337
 - zheng-chuang
+<<<<<<< HEAD
 - zxTomw
+=======
+
+>>>>>>> 83cf8b8b8 (add to contributors.yml)
```

```suggestion
- zxTomw
```

Fixing some merge cruft.

---

Whoops 😬